jhu-idc/idc-isle-dc
# Bumps the hash for idc_defaults to pull in the fix for the constraint on the field_unique_id

The fix itself is in the idc_defaults module.  It you'd like to review that it can be found here:  https://github.com/jhu-idc/idc_defaults/commit/6123bbb5e1f320761eafcd633b98ff79b33b1927

This fixes: https://github.com/jhu-idc/idc-isle-dc/issues/197 where the constraint for unique ids was not considering an update scenario. 

The constraint in the idc_defaults module will check to see if any entity type which has a field_unique_id has the same id as the object that's being saved. So it now considers if the object that was found has the same id (nid, mid, tid) as the item you're working with. If that's the case then it knows it's an update and will not create a violation. 

### To test this ticket
Reset to this branch and add a node item (either collection or RI, or try both).  See that the edit page has a filled in `Unique Id` for the item.  Fill in other required fields and click on save.  Then edit the item changing the name or something like that.   Save again.  It should save just fine (unlike what the tip of dev will do right now). 

Another test is create another node item. Try to give it the same unique id as the first one and ensure that it fails, with an error message about how the unique id is not unique.
